### PR TITLE
modules/nixos/monitoring: add grafana

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: modules/nixos/monitoring/grafana.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ config, ... }:
+{
+  services.grafana = {
+    enable = true;
+    settings = {
+      analytics.reporting_enabled = false;
+
+      "auth.anonymous".enabled = true;
```
Comment (Mic92) [inline modules/nixos/monitoring/grafana.nix:8]: We could use the github org for authentication here. Than we don't need to create accounts manually.

File: modules/nixos/monitoring/grafana.nix [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    ensureDatabases = [ "grafana" ];
+    ensureUsers = [
+      {
+        name = "grafana";
+        ensurePermissions = { "DATABASE grafana" = "ALL PRIVILEGES"; };
+      }
+    ];
+  };
+
+  services.postgresqlBackup = {
```
Comment (zowoq) [inline modules/nixos/monitoring/grafana.nix:58]: I'll open another PR for backups after this is merged.

File: modules/nixos/monitoring/grafana.nix [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+{
+  systemd.services.grafana.after = [ "prometheus.service" ];
+
+  services.grafana = {
+    enable = true;
+    settings = {
+      analytics.reporting_enabled = false;
+      analytics.feedback_links_enabled = false;
+
+      "auth.anonymous".enabled = false;
```
Comment (zowoq) [inline modules/nixos/monitoring/grafana.nix:11]: Don't see much point in making this public, only really of interest to nix-community org members.

Comment (zowoq) [reply modules/nixos/monitoring/grafana.nix]: We will actually want this public for logs from nixpkgs-updates with loki/promtail so I'll just leave it enabled.

**Pull request merged**